apps/nextjs/src/app/[locale]/boards/[name]/settings/_background.tsx

```diff
@@ -85,6 +127,13 @@ export const BackgroundSettingsContent = ({ board }: Props) => {
   );
 };
 
+const renderAutocompleteOption: AutocompleteProps["renderOption"] = ({ option }) => (
+  <Group gap="sm">
+    <img src={option.value} alt="preview image" style={{ width: 20, height: 20, objectFit: "contain" }} />
```

Comment:
Can you suppress the lint warnings? We scale the image on the backend, right? Therefore no component usage required